src/app/view/perspectives/search/documents/search-documents.component.html

```diff
@@ -1,11 +1,11 @@
-<div class="mt-2" *ngIf="documents">
-  <div *ngIf="documents.length == 0" class="alert alert-warning" i18n="@@search.result.empty">
+<div class="mt-2">
+  <div *ngIf="!((documents$ | async)?.length)" class="alert alert-warning" i18n="@@search.result.empty">
```

Comment:
You can do once `documents$ | async; let documents` and use documents from then on.